### ISSUE TITLE
Can't build cl-kraken/tests - missing package

### ISSUE THREAD

**Issue by quicklisp** [jonatack/cl-kraken#45]
This line triggers a build error for me because of a missing package: `Package CL-KRAKEN/SRC/TIME does not exist.`

https://github.com/jonatack/cl-kraken/blob/a14d27ad47726049484831c78440bdf6d655e303/tests/time.lisp#L9

**Comment by jonatack** [comment]
Thanks. I'm not sure how to reproduce this. The tests all pass locally and also when run by Travis CI, and the packages have not changed since 8 months or more.

**Comment by quicklisp** [comment]
I can reproduce it like so:

```
$ cd ~/quicklisp/local-projects
$ git clone https://github.com/jonatack/cl-kraken.git
```
Then in a fresh repl:
```
> (ql:quickload "cl-kraken/tests")
```

This isn't a new problem, I'm just newly reporting it.

**Comment by jonatack** [comment]
Thank you, reproduced. My .sbclrc and other .lisprc files were loading `(ql:quickload :cl-kraken)` which was hiding the issue. Will look into a fix.

**Comment by jonatack** [comment]
`(asdf:test-system :cl-kraken)` works, but `(rove:run :cl-kraken/tests)` and ` (ql:quickload "cl-kraken/tests")` do not. TIL, thanks again.

**Comment by jonatack** [comment]
Thank you again @quicklisp for reporting the issue. It should be closed by 022f2d0231757061175b62d4b8cc4cddbe871b5b.
